views/js/payment-components/stored-payment-method.js

```diff
@@ -46,10 +46,8 @@ jQuery(function ($) {
     // Iterate through the stored payment methods list we got from the adyen checkout component
     checkoutStoredPaymentMethods.forEach(function (storedPaymentMethod) {
 
-        /*
-            storedPaymentMethod.id = $storedPaymentApiId in the stored-payment-method.tpl
-            if the container doesn't exits don't try to render the component
-        */
+        //  storedPaymentMethod.id = $storedPaymentApiId in the stored-payment-method.tpl
+        //  if the container doesn't exits don't try to render the component
```

Comment:
remove this or make it the comment clear.

Reply:
removed, not necessary